Prevents the error about "please add -DCMAKE_POLICY_VERSION_MINIMUM=3.5"

--- a/xmake.lua
+++ b/xmake.lua
@@ -32,7 +32,7 @@ local moe_includedirs = {
 
 add_repositories("moe-repo xmake")
 
-local DOCTEST_VERSION = "2.4.11"
+local DOCTEST_VERSION = "2.4.12"
 
 add_requires("lolly")
 local tbox_configs = {hash=true, ["force-utf8"]=true, charset=true}
